x/gateway/module/abci.go

```diff
@@ -0,0 +1,21 @@
+package gateway
+
+import (
+	cosmostelemetry "github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/pokt-network/poktroll/x/gateway/keeper"
+	"github.com/pokt-network/poktroll/x/gateway/types"
+)
+
+// EndBlocker is called every block and handles gateway related updates.
+func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
```

Comment:
Can you look at `x/proof/module/abci.go` and add two things:
1. Return something from "UnbondGateways" (e.g. num gateways unbonded)
2. Log the value above
3. Add a log on errorr